pyaedt/modeler/cad/Primitives.py

```diff
@@ -4924,6 +4924,70 @@ def import_spaceclaim_document(self, input_file):
         self.refresh_all_ids()
         return True
 
+    def discovery_import(self, input_file):
+        """Import a Discovery file.
+
+        Parameters
+        ----------
+        input_file :
+            Full path and name of the Discovery file.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oEditor.CreateUserDefinedModel
+        """
+        version = settings.aedt_version[-4:].replace(".", "")
+
+        aedt_install_dir = os.environ.get("ANSYSEM_ROOT{}".format(version), "")
+        ansys_install_dir = os.environ.get("AWP_ROOT{}".format(version), "")
+
+        if ansys_install_dir == "":
+            self.logger.error("Same version of AEDT and Discovery is needed.")
+            return False
+
+        self.oeditor.CreateUserDefinedModel(
+            [
+                "NAME:UserDefinedModelParameters",
+                [
+                    "NAME:Definition",
+                    [
+                        "NAME:UDMParam",
+                        "Name:=",
+                        "GeometryFilePath",
+                        "Value:=",
+                        '"' + input_file + '"',
+                    ],
+                ],
+                [
+                    "NAME:Options",
+                    [
+                        "NAME:UDMParam",
+                        "Name:=",
+                        "IsDiscoveryLink",
+                        "Value:=",
+                        "1",
+                    ],
+                ],
+                ["NAME:GeometryParams"],
+                "DllName:=",
+                "SACADIntegUDM",
+                "Library:=",
+                "installLib",
+                "Version:=",
+                "1.0",
+                "ConnectionID:=",
+                "",
+            ]
+        )
+        self.refresh_all_ids()
```

Comment:
Since the method implement only the static import, I would break the link immediately after the import. and before refreshing all the IDs.